toolkit/tools/imagegen/diskutils/encryption.go

```diff
@@ -173,8 +173,16 @@ func encryptRootPartition(partDevPath string, partition configuration.Partition,
 		return
 	}
 
+	mkfsOptions, ok := DefaultMkfsOptions[partition.FsType]
+	if !ok {
```

Comment:
If it's not ok and someone is requesting to encrypt the drive, should this really just clear the options and continue?

Reply:
This was `_, stderr, err = shell.Execute("mkfs", "-t", partition.FsType, fullMappedPath)` before, with no options being passed at all. It will continue as it used to, if partition type key is not available in that map.